src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -2,13 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 import React from 'react';
 import { act, render } from '@testing-library/react';
+import { describeEachAppLayout, singleDrawer } from './utils';
 import AppLayout from '../../../lib/components/app-layout';
```

Comment:
This is the root cause of the issue. `./utils` file should be imported before app layout to initialize the module mock.

Thanks to the new test, it will be now captured in builds

Reply:
Non-blocking: What are the chances that a new test file is added and someone imported these utils after the AppLayout module? Can we add maybe a comment or so for these utils to point out this?

Reply:
ahh, I see now, looks good to me.

I saw the mock test you added but I overlooked how the `describeEachAppLayout` and `describeEachThemeAppLayout`  and that it will have that test running each time the utility is used, my bad.